src/stream/src/common/table/state_table.rs

```diff
@@ -140,6 +141,7 @@ where
     op_consistency_level: StateTableOpConsistencyLevel,
 
     clean_watermark_index_in_pk: Option<i32>,
+    clean_watermark_index_in_value: Option<i32>,
```

Comment:
I think we only need to store 1 of them as `clean_watermark_index_in_value` is actually a superset of `clean_watermark_index_in_pk` on semantics.